### PR TITLE
Modify animals

### PR DESCRIPTION
**Pull request by huseyinbiyik**
You can find the features added for this Pull Request below.

- Added a column species of type string to your animals table. Modify your schema.sql file.
Screen Shot 2022-07-19 at 21 02 16
- Added new animals/pokemons to the table.
- Inside a transaction  the animals table updated by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.↓
Screen Shot 2022-07-19 at 16 28 02
Screen Shot 2022-07-19 at 16 31 04

- Updated the animals table by setting the species column to digimon for all animals that have a name ending in mon.↓
Screen Shot 2022-07-19 at 16 41 08

- Updated the animals table by setting the species column to pokemon for all animals that don't have species already set.↓
Screen Shot 2022-07-19 at 16 45 53

- Commit the transaction.↓
Screen Shot 2022-07-19 at 16 57 24

- Verify that change was made and persists after commit. ↓
Screen Shot 2022-07-19 at 16 57 24

- Inside a transaction delete all records in the animals table, then roll back the transaction.↓
Screen Shot 2022-07-19 at 17 10 33

- After the rollback verify if all records in the animals table still exists.↓
Screen Shot 2022-07-19 at 17 11 04

- Delete all animals born after Jan 1st, 2022.↓
Screen Shot 2022-07-19 at 17 14 25

- Create a savepoint for the transaction.↓
Screen Shot 2022-07-19 at 17 15 47

- Update all animals' weight to be their weight multiplied by -1.↓
Screen Shot 2022-07-19 at 17 18 15

- Rollback to the savepoint↓
Screen Shot 2022-07-19 at 17 19 09

- Update all animals' weights that are negative to be their weight multiplied by -1. ↓
Screen Shot 2022-07-19 at 17 22 11

- Commit transaction↓
Screen Shot 2022-07-19 at 17 22 38

- How many animals are there?
Screen Shot 2022-07-19 at 17 26 32

- How many animals have never tried to escape?
Screen Shot 2022-07-19 at 17 48 17

- What is the average weight of animals?
Screen Shot 2022-07-19 at 17 52 29

- Who escapes the most, neutered or not neutered animals?
Screen Shot 2022-07-19 at 18 04 04

- What is the minimum and maximum weight of each type of animal?
Screen Shot 2022-07-19 at 18 13 33

- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
Screen Shot 2022-07-19 at 19 24 37

Thank you for the review in advance!🎉

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,10 +1,9 @@
 /* Database schema to keep the structure of entire database. */
-
-vet_clinic=# CREATE TABLE animals(
-id integer generated by default as identity
-(start with 1 increment by 1),
+vet_clinic = # CREATE TABLE animals(
```
Comment (mateo951) [inline schema.sql:2]: Please be sure that you use 

`ALTER TABLE animals ADD species varchar(255);` 

in order to add species column into your animal table 🤓

File: data.sql [hunk 2]
```diff
@@ -1,6 +1,121 @@
...
+insert into
+  animals(
+    name,
+    date_of_birth,
+    escape_attemps,
+    neutered,
+    weight_kg
+  )
+values
+  ('Blossom', '1998-10-13', 3, true, 17);
```
Comment (mateo951) [inline data.sql:110]: Propper data was introduced 😎

File: queries.sql [hunk 3]
```diff
@@ -1,10 +1,108 @@
...
-
-SELECT * from animals where name like '%mon';
-SELECT * from animals where date_of_birth between '2016-01-01' and '2019-01-01';
-SELECT * from animals where neutered = TRUE and escape_attemps < 3; 
-SELECT date_of_birth from animals where name in ('Agumon', 'Pikachu');
-SELECT name, escape_attemps from animals where weight_kg > 10.5;
-SELECT * from animals where neutered = true;
-SELECT * from animals where name NOT in ('Gabumon');
-SELECT * from animals WHERE weight_kg >=10.4 and weight_kg <=17.3;
+/*Queries that provide answers to the questions FROM all projects.*/
```
Comment (mateo951) [inline queries.sql:1]: Well done making all of the transactions 🙌

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @huseyinbiyik, 

🔥 Highlights 🔥
- Transactions 💪😎
- Answered all of the questions ✔

Splendid job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: schema.sql (hunk 1 above)
Comment (huseyinbiyik) [reply schema.sql:2]: Added now, thank you!
Screen Shot 2022-07-19 at 21 02 16

**Review by Deepakdanger — APPROVED** [review]
Hi @huseyinbiyik, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by huseyinbiyik** [comment]
Thank you. I will merge it now! 🤓

**Pull request merged**